Build of focal-builder container is failing

## Description

It appears the dependencies for dh-virtualenv cannot be resolved:
```
Step 10/12 : RUN apt-get update && apt-get install -y dh-virtualenv
 ---> Running in 262c816129d7
Hit:1 http://archive.ubuntu.com/ubuntu focal InRelease
Get:2 https://deb.debian.org/debian unstable InRelease [153 kB]
Hit:3 http://security.ubuntu.com/ubuntu focal-security InRelease
Hit:4 http://archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:5 http://archive.ubuntu.com/ubuntu focal-backports InRelease
Get:6 https://deb.debian.org/debian unstable/contrib amd64 Packages [73.2 kB]
Get:7 https://deb.debian.org/debian unstable/main amd64 Packages [11.7 MB]
Fetched 11.9 MB in 5s (2552 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 dh-virtualenv : Depends: libjs-sphinxdoc (>= 2.4.3-5~) but 1.8.5-7ubuntu3 is to be installed
                 Depends: sphinx-rtd-theme-common (>= 0.5.0+dfsg) but 0.4.3+dfsg-3 is to be installed
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-get update && apt-get install -y dh-virtualenv' returned a non-zero code: 100
make: *** [Makefile:7: build-container] Error 100

```
## Steps to Reproduce
- cd molecule/builder-focal
- make build-container
- observe error

## Expected Behavior
- Container should build build successfully

## Actual Behavior
- Container fails to build

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -296,13 +296,13 @@ jobs:
           name: Check Python dependencies for CVEs
           command: |
             fromtag=$(docker images |grep securedrop-test-xenial-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "sudo pip3 install -q --upgrade safety && make -C .. safety"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade safety && make -C .. safety"
 
       - run:
           name: Run static security testing on source code
           command: |
             fromtag=$(docker images |grep securedrop-test-xenial-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "sudo pip3 install -q --upgrade pip && sudo pip3 install -q --upgrade bandit && make -C .. bandit"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade bandit && make -C .. bandit"
 
 
   staging-test-with-rebase:

--- a/molecule/builder-focal/Dockerfile
+++ b/molecule/builder-focal/Dockerfile
@@ -39,10 +39,8 @@ RUN apt-get -y update && apt-get upgrade -y && apt-get install -y \
 
 # TEMPORARY: install dh-virtualenv from debian unstable, pending focal package:
 # https://github.com/spotify/dh-virtualenv/issues/298
-RUN echo "deb https://deb.debian.org/debian unstable main contrib" > /etc/apt/sources.list.d/debian-unstable.list
-COPY aptpreferences.conf /etc/apt/preferences.d/debian-unstable
-RUN apt-get install -y debian-archive-keyring
-RUN ln -s /usr/share/keyrings/debian-archive-keyring.gpg /etc/apt/trusted.gpg.d/
+RUN echo "deb http://archive.ubuntu.com/ubuntu/ groovy universe" > /etc/apt/sources.list.d/ubuntu-groovy.list
+COPY aptpreferences.conf /etc/apt/preferences.d/ubuntu-groovy
 
 RUN apt-get update && apt-get install -y dh-virtualenv
 

--- a/molecule/builder-focal/aptpreferences.conf
+++ b/molecule/builder-focal/aptpreferences.conf
@@ -3,5 +3,5 @@ Pin: release a=focal
 Pin-Priority: 700
 
 Package: *
-Pin: release a=unstable
+Pin: release a=groovy
 Pin-Priority: 1

--- a/molecule/builder-focal/image_hash
+++ b/molecule/builder-focal/image_hash
@@ -1,2 +1,2 @@
-# sha256 digest quay.io/freedomofpress/sd-docker-builder-focal:2020_09_11
-10d4b7f28a3de6f2bd9e6a55dd5fdb448fb90442d15a013f73942b4d2e5ce259
+# sha256 digest quay.io/freedomofpress/sd-docker-builder-focal:2021_01_26
+60436817a1e7a2b1f2abe19ef456b73e0b6e6d4064f2edb27e1ae0da3fcccef3

--- a/molecule/builder-xenial/image_hash
+++ b/molecule/builder-xenial/image_hash
@@ -1,2 +1,2 @@
-# sha256 digest quay.io/freedomofpress/sd-docker-builder-xenial:2020_10_07
-35b70776cf5139ed2c597063579d74f0458416e4cbbe443bfebef2596ce5a48c
+# sha256 digest quay.io/freedomofpress/sd-docker-builder-xenial:2021_01_26
+f17105ea2618621d2de42ee6ef4d2cff507845fbf4d26581581bcf7c99d47217

